spring-integration-jmx/src/main/java/org/springframework/integration/monitor/IntegrationMBeanExporter.java

```diff
@@ -111,13 +117,13 @@
  */
 @ManagedResource
 public class IntegrationMBeanExporter extends MBeanExporter implements BeanPostProcessor, BeanFactoryAware,
```

Comment:
Because the `IMBE` doesn't export itself.

http://docs.spring.io/spring-integration/reference/html/system-management-chapter.html#jmx-mbean-exporter